io_scene_godot/converters/grease.py

```diff
@@ -0,0 +1,367 @@
+        #                    values=[v.normal for v in self.vertices])
+
+        surface_lines.append(position_vals.to_string())
+        #surface_lines.append(normal_vals.to_string())
+        surface_lines.append("null, ; No Normals")
+        surface_lines.append(self.get_tangent_array().to_string())
+        surface_lines.append(self.get_color_array().to_string())
+
+        surface_lines.append(self.get_uv_array(0).to_string())
+        surface_lines.append(self.get_uv_array(1).to_string())
+
+        surface_lines.append("null, ; No Bones")
+        surface_lines.append("null, ; No Bone Weights")
+
+
+        # Indices- each face is made of 3 verts, and these are the indices
+        # in the vertex arrays. The backface is computed from the winding
+        # order, hence v[2] before v[1]
+        if self.indices:
+            face_indices = Array(
+                "IntArray(",
+                #values=[[v[0], v[2], v[1]] for v in self.indices]
+                flat_values = self.indices
+            )
+        else:
+            # in morph, it has no indices
+            face_indices = Array(
+                "null, ; Morph Object", "", ""
+            )
+
+        surface_lines.append(face_indices.to_string())
+
+        return surface_lines
+
+    def to_string(self):
+        """Serialize"""
+        return self.generate_lines().to_string()
+
+
+class Surface:
+    """A surface is a single part of a mesh (eg in blender, one mesh can have
+    multiple materials. Godot calls these separate parts separate surfaces"""
+
+    def __init__(self):
+        # map from a Vertex.tup() to surface.vertex_data.indices
+        self.vertex_map = dict()
+        self.vertex_data = VerticesArrays()
+        self.morph_arrays = Array(prefix="[", seperator=",\n", suffix="]")
+        # map from mesh.loop_index to surface.vertex_data.indices
+        self.vertex_index_map = dict()
+        self.id = None
+        self.material = None
+
+    @property
+    def name_str(self):
+        """Used to separate surfaces that are part of the same mesh by their
+        id"""
+        return "surfaces/" + str(self.id)
+
+    def generate_object(self):
+        """Generate object with mapping structure which fully
+        describe the surface"""
+        surface_object = Map()
+        if self.material is not None:
+            surface_object['material'] = self.material
+        #surface_object['primitive'] = 5  # triangle strip, sucks with complex shapes
+        #surface_object['primitive'] = 6  # triangle fan, looks better than strip
+        surface_object['primitive'] = 3  # simpler to debug
+        surface_object['arrays'] = self.vertex_data
+        surface_object['morph_arrays'] = self.morph_arrays
+        return surface_object
+
+    def to_string(self):
+        """Serialize"""
+        return self.generate_object().to_string()
+
+# PRIMITIVE_POINTS = 0 — Render array as points (one vertex equals one point).
```

Comment:
Instead of making them all comments, maybe you can define it as global variable, it would be more readable when use them in the code.

e.g.
```
# PRIMITIVE_POINTS = 0 — Render array as points (one vertex equals one point).
```

Could be changed to 
```
PRIMITIVE_POINTS = 0  # Render array as points (one vertex equals one point).
```
And put them into `utils.py` so that they can be shared with `mesh.py`